web/src/pages/support/login.tsx

```diff
@@ -0,0 +1,107 @@
+import { Content } from "@/components/Content";
+import { Heading } from "@/components/njwds-extended/Heading";
+import { PrimaryButton } from "@/components/njwds-extended/PrimaryButton";
+import { UnStyledButton } from "@/components/njwds-extended/UnStyledButton";
+import { PageSkeleton } from "@/components/njwds-layout/PageSkeleton";
+import { SingleColumnContainer } from "@/components/njwds/SingleColumnContainer";
+import { AuthContext } from "@/contexts/authContext";
+import { IsAuthenticated } from "@/lib/auth/AuthContext";
+import { configureAmplify, triggerSignOut } from "@/lib/auth/sessionHelper";
+import { useConfig } from "@/lib/data-hooks/useConfig";
+import analytics from "@/lib/utils/analytics";
+import { NextSeo } from "next-seo";
+import { ReactElement, useContext } from "react";
+
+const LoginSupportPage = (): ReactElement => {
+  const { Config } = useConfig();
+  const loginSupport = Config.loginSupportPage;
+  const { state: authState } = useContext(AuthContext);
+
+  const isLoggedIn = authState.isAuthenticated === IsAuthenticated.TRUE;
+
+  const customComponents = {
+    logoutButton: (
+      <UnStyledButton
+        onClick={async () => {
+          configureAmplify();
+          await triggerSignOut();
+        }}
+        className="logout-button-unstyled"
+      >
+        log out
+      </UnStyledButton>
+    ),
+  };
+
+  return (
+    <>
+      <NextSeo
+        title={loginSupport.pageTitle}
+        description="Step-by-step guide to help you log in to Business.NJ.gov using your myNewJersey account."
+        noindex={true}
+      />
+      <PageSkeleton showNavBar isSeoStarterKit={!isLoggedIn} isLoginPage={isLoggedIn}>
```

Comment:
This is the least additive way to conditionally render the nav bar in the varying states we want it to show up, which is:
1. if a user is _not_ logged in, show the log in button
2. if a user _is_ logged in, the nav bar should _hide_ the login button.

There's [an existing ticket](https://dev.azure.com/NJInnovation/Business%20First%20Stop/_queries/edit/13005/?queryId=9275a23f-760c-4ab8-8178-6f29a87f5b39) to refactor the nav bar components into something utilizing enums and different nav bar types, rather than the proliferation of various boolean flags that handle different variations of the nav bar. I opted to keep the scope leaner here and delegate some of the nav bar cleanup to that ticket